mock-mhs-adaptor/src/main/java/uk/nhs/adaptors/mockmhsservice/common/OutboundMessage.java

```diff
@@ -30,6 +30,7 @@
     @Setter
     @Jacksonized
     @AllArgsConstructor
+    @Builder
```

Comment:
optional, other places this is used we could change from constructor to builder

Reply:
in mock mhs we don't create OutboundMessage, only parsing it, but I was not able to build the service if the annotation was nat provided.